### PR TITLE
Create calendar markup + styles

### LINKED ISSUE #7: Create Calendar markup

### PR DESCRIPTION
**Pull request by al-ros**

### CODE REVIEW

File: src/components/CalendarMarkup/CalendarMarkup.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+const CalendarMarkup = () => {
+    const { user } = useContext(UsersContext)
+    const days = ['Mon', 'Tue', 'Wed', 'Thu', 'Fri']
+    const times = ['10:00', '11:00', '12:00', '13:00', '14:00', '15:00', '16:00', '17:00', '18:00']
+    return(
+        <div className="table">
+            <div className="table__row">
+                <div className="table__cell table__cell--head">{user?.label}</div>
+                {days.map((day) => <div className="table__cell table__cell--head">{day}</div>)}
```
Comment (roman-vabishchevych) [inline src/components/CalendarMarkup/CalendarMarkup.jsx]: add attribute `key` for each map()
```suggestion
                {days.map((day) => <div key={ day } className="table__cell table__cell--head">{day}</div>)}
```

File: src/App.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import Header from './components/Header';
 import ModalAuth from './components/ModalAuth';
 import UsersContext from './contexts/UsersContext';
 import './App.css';
+import CalendarMarkup from './components/CalendarMarkup';
```
Comment (roman-vabishchevych) [inline src/App.js]: Name the component `Calendar`

File: src/components/CalendarMarkup/CalendarMarkup.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { useContext } from 'react'
+import './CalendarMarkup.scss'
+import UsersContext from '../../contexts/UsersContext'
+
+const CalendarMarkup = () => {
+    const { user } = useContext(UsersContext)
+    const days = ['Mon', 'Tue', 'Wed', 'Thu', 'Fri']
+    const times = ['10:00', '11:00', '12:00', '13:00', '14:00', '15:00', '16:00', '17:00', '18:00']
+    return(
+        <div className="table">
```
Comment (roman-vabishchevych) [inline src/components/CalendarMarkup/CalendarMarkup.jsx]: use block className as component named: `Calendar -> className="calendar"`

File: src/components/CalendarMarkup/CalendarMarkup.jsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { useContext } from 'react'
+import './CalendarMarkup.scss'
+import UsersContext from '../../contexts/UsersContext'
+
+const CalendarMarkup = () => {
+    const { user } = useContext(UsersContext)
+    const days = ['Mon', 'Tue', 'Wed', 'Thu', 'Fri']
```
Comment (roman-vabishchevych) [inline src/components/CalendarMarkup/CalendarMarkup.jsx]: Please pull the last changes from 'main' and move days and times to the /src/constants.js

**Comment by al-ros** [comment]
All changes resolved

**Review by roman-vabishchevych — APPROVED** [review]

**Pull request merged**